AmericanExpress/src/main/java/com/braintreepayments/api/AmericanExpressClient.java

```diff
@@ -49,21 +49,21 @@ public void getRewardsBalance(@NonNull String nonce, @NonNull String currencyIso
                 .build()
                 .toString();
 
-        braintreeClient.sendAnalyticsEvent("amex.rewards-balance.start");
+        braintreeClient.sendAnalyticsEvent(AmericanExpressAnalytics.REWARDS_BALANCE_STARTED);
         braintreeClient.sendGET(getRewardsBalanceUrl, (responseBody, httpError) -> {
             if (responseBody != null) {
-                braintreeClient.sendAnalyticsEvent("amex.rewards-balance.success");
                 try {
                     AmericanExpressRewardsBalance rewardsBalance =
                             AmericanExpressRewardsBalance.fromJson(responseBody);
                     callback.onAmericanExpressResult(new AmericanExpressResult.Success(rewardsBalance));
+                    braintreeClient.sendAnalyticsEvent(AmericanExpressAnalytics.REWARDS_BALANCE_SUCCEEDED);
                 } catch (JSONException e) {
-                    braintreeClient.sendAnalyticsEvent("amex.rewards-balance.parse.failed");
+                    braintreeClient.sendAnalyticsEvent(AmericanExpressAnalytics.REWARDS_BALANCE_FAILED);
                     callback.onAmericanExpressResult(new AmericanExpressResult.Failure(e));
                 }
             } else if (httpError != null) {
                 callback.onAmericanExpressResult(new AmericanExpressResult.Failure(httpError));
-                braintreeClient.sendAnalyticsEvent("amex.rewards-balance.error");
+                braintreeClient.sendAnalyticsEvent(AmericanExpressAnalytics.REWARDS_BALANCE_FAILED);
```

Comment:
I know in other modules we are refactoring to have a single `success` and `error` method for callbacks and analytics to confirm we always send analytics with the callback. That seemed a little overkill in this module since there is only one method, but I can add those in if we think it's still beneficial

Reply:
We did it on iOS, just for consistency - but whatever you think works on Android 👍

Reply:
Updated for consistency too 👍